src/ukify/ukify.py

```diff
@@ -453,6 +454,14 @@ def make_uki(opts):
 
     uki = UKI(opts.stub)
 
+    match opts.initrd:
+        case []:
+            initrd = None
+        case [initrd]:
+            pass
+        case multiple:
+            initrd = b''.join(file.open('rb').read() for file in multiple)
```

Comment:
you must pad each initrd to multiples of 32bit, with NUL bytes. (except for the last one)

Reply:
also, since the elements of `multiple` should be `Path` objects, this can be made a bit shorter:
```suggestion
            initrd = b''.join(file.read_bytes() for file in multiple)
```